lib/src/stack/layers/blockwise_layer.dart

```diff
@@ -10,26 +10,25 @@ import 'dart:math';
 
```

Comment:
This file could perhaps be named just `blockwise.dart`, I would guess that it's a layer anyway since it's implied by the enclosing folder

Reply:
Good point :) Should we remove the `layer` part from all files in the folder?

Reply:
Renamed all the files in 92fa81cdc336ed21898e963f00c22178ccff3f0c for now